hook/hook.go

```diff
@@ -381,6 +381,7 @@ type Hook struct {
 	PassEnvironmentToCommand            []Argument      `json:"pass-environment-to-command,omitempty"`
 	PassArgumentsToCommand              []Argument      `json:"pass-arguments-to-command,omitempty"`
 	JSONStringParameters                []Argument      `json:"parse-parameters-as-json,omitempty"`
+	MaxConcurrency                      int             `json:"max-concurrency,omiempty"`
```

Comment:
Typo in `omiempty` -> missing letter _t_ after _i_

Reply:
I would rename the flag to `maximum-concurrent-executions`

Reply:
fixed